dib/tag.go

```diff
@@ -25,18 +27,33 @@ func Retag(graph *dag.DAG, tagger types.ImageTagger, oldTag string, newTag strin
 	})
 }
 
-// RetagLatest iterates over the graph to retag each image with the latest tag.
-func RetagLatest(graph *dag.DAG, tagger types.ImageTagger, tag string) error {
+// AddExtraTags iterates over the graph to retag each image with the latest tag.
```

Comment:
I think the `Add` prefix is misleading, can we keep `Retag` or `Tag` ?
This would look better like `TagWithExtraTags` or something

Then, don"t forget to update the comment to remove the reference to `latest`